string  name
float64 value
string encoding
int32 state_space
bool include_in_rl
